src/cts/src/CtsOptions.h

```diff
@@ -67,13 +67,13 @@ class CtsOptions
     bufferList_ = buffers;
   }
   std::vector<std::string> getBufferList() const { return bufferList_; }
-  void setDbUnits(int units) { dbUnits_ = units; }
-  int getDbUnits() const { return dbUnits_; }
-  void setWireSegmentUnit(unsigned wireSegmentUnit)
+  void setDbUnits(float units) { dbUnits_ = units; }
```

Comment:
Why is this a float now?

Reply:
TechChar.cpp line 447 confused me that the value is actually a float but yes, it should be an int.

Reply:
Sorry, I confused it.